app/src/app/api/v0/inventory/[inventory]/route.ts

```diff
@@ -46,8 +44,7 @@ export const DELETE = apiHandler(async (_req, { params, session }) => {
 
 export const PATCH = apiHandler(async (req, context) => {
   const { params, session } = context;
-  const body = createInventoryRequest.parse(await req.json());
-
+  const body = upsertInventoryRequest.parse(await req.json());
```

Comment:
If `isPublic` is changed to true here, the publishedAt timestamp needs to be updated (only if it wasn't public before)